spec/workers/extract_worker_spec.rb

```diff
@@ -24,10 +24,10 @@
       end.not_to raise_error
     end
 
-    it 'reraises error if no extracts exist' do
+    it 'no longer raises error even if no extracts exist' do
```

Comment:
Should be "if no classification exists", not "if no extracts exist"

Reply:
true!

Reply:
or wait, actually, not quite: the code used to work differently when the classification was missing depending on whether or not any extracts had been produced, which is why it reads that way. i could have consolidated the test cases and removed the check but i wanted to be explicit about what i was doing.

Reply:
Oh yeah, needed to zoom out a bit. Those specs may not make perfect sense together since the expectations are the same now but this isn't blocking

Reply:
Maybe roll into Barn Door 2.0?

Reply:
yeah it's probably something to clean up later but i wanted this diff to be clearer :)